src/components/Tooltip/README.md

```diff
@@ -0,0 +1,47 @@
+# Tooltip
+
+```vue
+<Tooltip :fallback-placements="['bottom']">
+    Insert coin here
+</Tooltip>
+```
+
+## Imported Props
+
+| Prop name          | Description                                | Type                                                              | Values                                | Default                            |
+| ------------------ | ------------------------------------------ | ----------------------------------------------------------------- | ------------------------------------- | ---------------------------------- |
+| fallbackPlacements | The fallback placements of the component   | [TooltipFallbackPlacement](../../composables/useTooltipPlacement) | ['top'\|'bottom'\|'left'\|'right']    | ['top', 'right', 'bottom', 'left'] |
+| placement          | The default placement of the component     | [TooltipPlacement](../../composables/useTooltipPlacement)         | ['top'\|'bottom'\|'left'\|'right']    | 'top'                              |
```

Comment:
```suggestion
| placement          | The default placement of the component     | [TooltipPlacement](../../composables/useTooltipPlacement)         | ['top'\|'bottom'\|'left'\|'right'\|'auto']    | 'top'                              |
```

Reply:
It exists in `placement` under `Options` https://getbootstrap.com/docs/5.2/components/tooltips/

Reply:
I know, that's why I suggested to add it

Reply:
Ah, I added it somewhere else then. I know I added it somewhere... smh